source/app/views/new_game.erb

```diff
@@ -0,0 +1,24 @@
+<h2>Welcome To Rock, Paper, Scissors! </h2>
+
+
+<form action= '/game' method='post'>
+  <input type="hidden" name="player_1_id" value= '1'>
+  <input type="hidden" name="player_2_id" value= '2'>
+
+  <div id='player_1_input'>
+    <p>Player 1! Choose Wisely!</p>
```

Comment:
we can just have a loop here through all of the tokens and output their inputs and radios!